<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simple Youtube Downloader</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon/yt_logo.png</normaloff>icon/yt_logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #FEFCF3;</string>
  </property>
  <property name="iconSize">
   <size>
    <width>32</width>
    <height>32</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="yt_logo">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>icon/yt_logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="author_text">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>100</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Created by ArioNET35</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="link_yt">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>140</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
border-width: 5px;
border-color: #7FE9DE;</string>
    </property>
    <property name="placeholderText">
     <string>Paste link disini</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_getlink">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>140</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
border-width: 5px;
border-color: #7FE9DE;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icon/arrow.png</normaloff>icon/arrow.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_border">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>230</y>
      <width>391</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 5px;
border-style: solid;
border-color: #7FE9DE;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>250</y>
      <width>351</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <property name="labelAlignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="formAlignment">
      <set>Qt::AlignJustify|Qt::AlignVCenter</set>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="judul">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Judul</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="openExternalLinks">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="getjudul">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="durasi">
       <property name="text">
        <string>Durasi</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="getdurasi">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="views">
       <property name="text">
        <string>Views</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="getviews">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="details">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>210</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
     </font>
    </property>
    <property name="text">
     <string>Details</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dvideo">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>420</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #7FE9DE;
border-style: rounded;
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string>Download Video</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_getlokasi">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>370</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
border-width: 5px;
border-color: #7FE9DE;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icon/folder-horizontal-open.png</normaloff>icon/folder-horizontal-open.png</iconset>
    </property>
   </widget>
   <widget class="QLineEdit" name="link_lokasi">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>370</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
border-width: 5px;
border-color: #7FE9DE;</string>
    </property>
    <property name="placeholderText">
     <string>Pilih lokasi penyimpanan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="daudio">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>420</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #7FE9DE;
border-style: rounded;
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string>Download Audio</string>
    </property>
   </widget>
   <widget class="QLabel" name="lokasi">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>350</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
     </font>
    </property>
    <property name="text">
     <string>Pilih Lokasi Penyimpanan</string>
    </property>
   </widget>
   <widget class="QLabel" name="success">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>480</y>
      <width>351</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: green;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>yt_logo</zorder>
   <zorder>author_text</zorder>
   <zorder>btn_getlink</zorder>
   <zorder>link_yt</zorder>
   <zorder>label_border</zorder>
   <zorder>formLayoutWidget</zorder>
   <zorder>details</zorder>
   <zorder>dvideo</zorder>
   <zorder>btn_getlokasi</zorder>
   <zorder>link_lokasi</zorder>
   <zorder>daudio</zorder>
   <zorder>lokasi</zorder>
   <zorder>success</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>748</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAuthor"/>
   </widget>
   <widget class="QMenu" name="menuTheme">
    <property name="title">
     <string>Theme</string>
    </property>
    <addaction name="actionLight"/>
    <addaction name="actionDark"/>
    <addaction name="actionPDIP"/>
   </widget>
   <addaction name="menuAbout"/>
   <addaction name="menuTheme"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAuthor">
   <property name="text">
    <string>Author</string>
   </property>
  </action>
  <action name="actionLight">
   <property name="text">
    <string>Light</string>
   </property>
  </action>
  <action name="actionDark">
   <property name="text">
    <string>Dark</string>
   </property>
  </action>
  <action name="actionPDIP">
   <property name="text">
    <string>PDIP</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
